src/registry/textarea/preview.tsx

```diff
@@ -0,0 +1,48 @@
+import {TextareaComponentSchema} from '@open-formulieren/types';
+
+import {TextArea} from '@/components/formio';
+
+import {ComponentPreviewProps} from '../types';
+
+/**
+ * Show a formio textarea component preview.
+ *
+ * NOTE: for the time being, this is rendered in the default Formio bootstrap style,
+ * however at some point this should use the components of
+ * @open-formulieren/formio-renderer instead for a more accurate preview.
+ */
+const Preview: React.FC<ComponentPreviewProps<TextareaComponentSchema>> = ({component}) => {
+  const {
+    key,
+    label,
+    description,
+    placeholder,
+    tooltip,
+    validate = {},
+    autocomplete = '',
+    disabled = false,
+    multiple,
+    showCharCount,
+    rows = 3,
```

Comment:
I wouldn't give this a default value since it can be left empty, no?

Reply:
Types also mark it as required, but I think it shouldn't be in this case?

Reply:
especially if the type marks it required I wouldn't expect a default value in the destructuring, but following https://github.com/open-formulieren/formio-builder/pull/58#discussion_r1400788984 we should be able to just pass it along whether a value is set or not (undefined) without a default value.

Reply:
https://github.com/open-formulieren/types/pull/28

Reply:
Merged & release tag pushed, it should be on npm in a couple of minutes